src/MoviePage/index.js

```diff
@@ -29,6 +23,11 @@ function MoviePage() {
     }
   }
 
+  async function setMoviesData() {
+    const result = await getMovies(page);
+    setMovies(movies.concat(result.results));
```

Comment:
I think is not good to use hooks variables/functions outside of other hooks you should use the await inside other hooks

Reply:
Done!!